addons/web_editor/static/src/js/editor/odoo-editor/src/commands/deleteBackward.js

```diff
@@ -101,7 +101,7 @@ HTMLElement.prototype.oDeleteBackward = function (offset, alreadyMoved = false,
             parentEl.oid !== "root" &&
             parentEl.parentElement &&
             !parentEl.parentElement.isContentEditable &&
-            paragraphRelatedElements.includes(this.tagName) &&
+            this.tagName === 'P' &&
```

Comment:
I wrote a comment on the task about this, I'm not sure about this spec. But if we keep it, this does seem like the right way to go about it.